live/engine/registry_state.py

```
@@ -0,0 +1,427 @@
+    stale = (seconds_since_poll is not None and seconds_since_poll > STALE_THRESHOLD_SEC)
+
+    # Idle check
+    has_resting = any(o["status"] in ("open", "pending", "partial") for o in orders_list)
+    fills_list = []
+    for f in fills_rows:
+        f_dict = dict(f)
+        f_dict["age_sec"] = max(0.0, round((now_ms - f_dict["venue_ts"]) / 1000.0, 1))
+        try:
+            dt = datetime.datetime.fromtimestamp(f_dict["venue_ts"] / 1000.0)
+            f_dict["venue_time_str"] = dt.strftime("%Y-%m-%d %H:%M:%S")
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)